api/src/auth/drivers/openid.ts

```diff
@@ -392,7 +396,9 @@ export class OpenIDAuthDriver extends LocalAuthDriver {
 
 				// Update user refreshToken if provided
 				if (tokenSet.refresh_token) {
-					await this.usersService.updateOne(user.id, {
+					const usersService = await this.getUsersService();
```

Comment:
```suggestion
					const usersService = this.getUsersService(schema);
```

Reply:
Ah i see the github collapsed view got me and `schema` wasnt already available. nice solve